Add document for schema loader (#43)
scalar-labs/scalardl: docs/schema-loader.md

> # scalardl-schema-loader

Reviewer: ```suggestion
# ScalarDL Schema Loader
```

> A Docker image that loads the database schema with [Schema Tool for Scalar DB](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/).

Reviewer: ```suggestion
A Docker image that loads the database schemas of ScalarDL using [Schema Tool for Scalar DB](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/).
```

> ## How to Build
> 
> ```console
> docker build . -t ghcr.io/scalar-labs/scalardl-schema-loader:<version>

Reviewer 2: ```suggestion
## How to Pull

```console
docker pull ghcr.io/scalar-labs/scalardl-schema-loader:<version>
```
I think the `scalardl` repository includes documents for users. And, basically, users cannot access the `scalardl-schema-loader` (i.e., they cannot build the container image themselves) since it is a private repository.

So, I think it is better to explain how to get (pull) the container image here. 
What do you think?

(Strictly, users need permission also to pull the container image. So, I think we should consider updating this document based on the Marketplaces in the future.)

Author: @kota2and3kan 

Ah, you are right!
Thank you for catching it.
Sorry, I didn't notice this part.
They are in different repositories now.

How about we remove the how "How to Build" (How to Pull) section first and add it back later when we have public releases for it?

Reviewer 2: @supl 

> How about we remove the how "How to Build" (How to Pull) section first and add it back later when we have public releases for it?

I see. That sounds good!
Let's remove the `How to Build` section now.

Author: @kota2and3kan 

Thank you for the confirmation.

I removed the section in https://github.com/scalar-labs/scalardl/pull/43/commits/9a15d750e03c372910220ddf82c524a11f58e823

Please take a look 🙇
